packages/fx-core/resource/package.nls.json

```diff
@@ -843,8 +843,8 @@
   "error.BadComponent": "Component '%s' lacks of property: %s.",
   "error.InvalidFeature": "This feature is not applicable for your project.",
   "error.UpdateAadManifest.MissingEnvHint": "If you are developing with a new project created with Teams Toolkit, running provision or debug will register correct values for these environment variables.",
-  "driver.aadApp.description.create": "Create an Azure Active Directory app to authenticate users",
-  "driver.aadApp.description.update": "Apply Azure Active Directory app manifest to an existing app",
+  "driver.aadApp.description.create": "Create an Azure Active Directory app to authenticate users.",
```

Comment:
authenticate or authenticated?

Reply:
Here the aad app is used to authenticate users of the app. So it should be 'authenticate'